sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveQl.scala

```diff
@@ -557,8 +557,11 @@ https://cwiki.apache.org/confluence/display/Hive/Enhanced+Aggregation%2C+Cube%2C
             "TOK_TABLEPROPERTIES"),
           children)
       val (db, tableName) = extractDbNameTableName(tableNameParts)
-
-      CreateTableAsSelect(db, tableName, nodeToPlan(query), allowExisting != None, Some(node))
+      if (ignores.exists(_ != None)) {
+        CreateTableAsSelect(db, tableName, nodeToPlan(query), allowExisting != None, Some(node))
```

Comment:
That's my bad I didn't drop a comment here.
Actually we are not using the `ignores`, previously we were trying the parse the storage format etc. information within HiveQl.scala, however, the logic is quite complicated and error-prone, and then we decided to reuse the Hive code for that purpose, that's why we always pass down the `node` (ASTNode) . Even without storage format specified, Hive still can provide the default storage format for that. So I don't think the change here is reasonable.